### PR TITLE
Feature/challenge component

### PR DESCRIPTION
**Pull request by MaryanaMysak**
Знімок екрана 2021-07-12 о 14 42 56
Знімок екрана 2021-07-12 о 14 43 26

### CODE REVIEW

File: src/helpers/getRandomImage.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  challenge4,
+  challenge5,
+  challenge6,
+  challenge7,
+  challenge8,
+];
+
+export const getRandomImage = () => {
+  const imagesAmount = challengeImages.length;
+  const randomIndex = Math.floor(Math.random() * imagesAmount);
```
Comment (StepanNaryshkovEleks) [inline src/helpers/getRandomImage.js:23]: the random image should be changed after refreshing the page.
it is an example from prev. project, but you can just modify it

```
export const GROUP_ID = -1;

export default function getRandomColor(id: number) {
  if (id === GROUP_ID) return '#317ad8';

  const lastDigit = id % 10;
  const colorsRange = ['#677d96', '#e169b1', '#eb8c57', '#f2b94a', '#3591c5', '#697ce1'];
  const arrIndex = lastDigit > colorsRange.length ? lastDigit - (colorsRange.length + 1) : lastDigit;
  return colorsRange[arrIndex];
}

```

File: src/components/challenge/_.module.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
+.challenge {
```
Comment (StepanNaryshkovEleks) [inline src/components/challenge/_.module.css:1]: image
do not have these problems with footer?

File: src/components/challenge/_.module.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  background-color: white;
```
Comment (StepanNaryshkovEleks) [inline src/components/challenge/_.module.css:2]: There is not an opportunity to see a description of the challenge on the screen.
image

File: src/components/challenge/component.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      </header>
+      <img src={challengeImg} alt="Challenge" className={styles.challengeImg} />
+      <Link to={CNST.ROUTES.HOME}>
+        <div className={styles.footer}>
+          <div className={styles.votesContainer}>
+            <span className={styles.id}>#100245</span>
+            {!isOpen && (
+              <>
+                <img src={userImg} alt="Winner" className={styles.winnerImg} />
+                <span className={styles.winnerName}>Loulou won the challenge</span>
```
Comment (StepanNaryshkovEleks) [inline src/components/challenge/component.js:30]: we can add 3 dots for the name it is a long name
image

**Comment by MaryanaMysak** [comment]
comments fixed

**Pull request merged**